tests/test_definitions/multicolumn_map_expectations/expect_compound_columns_to_be_unique.json

```diff
@@ -240,21 +286,14 @@
       }
     },{
       "title": "unexpected_values_exact_match_out_with_unexpected_index_list",
-      "exact_match_out" : true,
```

Comment:
Why was this changed?

Reply:
This is because the JSON tests handle both v2 and v3 expectations. 
* `exact_match_out` ensures the entire dictionary matches expectations, but this can't be the case. 
* V3 will contain ID/PK related parameters in the `unexpected_index_list` while V2 will not